docs/test-surveys/13-survey-react-native-release.md

```diff
@@ -0,0 +1,87 @@
+### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city and a further on.
+
+#### Describe the steps you took.
+
+*Input box*
+
+### UC2: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Find out when shops are usually open in Regensburg. Switch the language as soon as you found it.
```

Comment:
You should probably mention the goal should be to find the information without using the search.